### DISCUSSION TITLE
Not able to see a step's logs while it is running

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#25450] [Actions]
<p>My step has been running for 25mins and I can’t see what it is doing:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/9/f9d1073ff66f3e150669acedaa160d36e1694b19.png" data-download-href="/uploads/short-url/zDYJkVGZEZhHuwtF9lJLNUgrgRr.png?dl=1" title="Selection_313" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/110988/181091174-3210a9e4-08da-4fe9-8d01-2fa2e765aff3.png" alt="Selection_313" data-base62-sha1="zDYJkVGZEZhHuwtF9lJLNUgrgRr" width="690" height="133" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/9/f9d1073ff66f3e150669acedaa160d36e1694b19_2_690x133.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/9/f9d1073ff66f3e150669acedaa160d36e1694b19_2_1035x199.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/9/f9d1073ff66f3e150669acedaa160d36e1694b19_2_1380x266.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/9/f9d1073ff66f3e150669acedaa160d36e1694b19_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Selection_313</span><span class="informations">1475×286 27.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This is a regression compared to other CI services. Is this known?</p>
<p>The action run is <a href="https://github.com/Mudlet/Mudlet/runs/1257604009" rel="noopener nofollow ugc">https://github.com/Mudlet/Mudlet/runs/1257604009</a>.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/vadi2">@vadi2</a>,</p>
<p>Yes, this is a known GitHub UI issue.</p>
<p>Sometimes we can view the real-time logs during the steps running, but sometimes we can’t expand and view the logs during the steps running or stuck until the steps or jobs completed.</p>
<p>Earlier, we had created the internal issue ticket to report the issue to the appropriate engineering team for further investigation and evaluation.</p>

**Reply by dsayling** [reply]
will this likely ever be fixed? I understand how it's not likely a priority, but it can be really helpful for monitoring.
